tc-cli/src/main.rs

```diff
@@ -123,7 +124,7 @@ enum Command {
 	},
 	RegisterShards,
 	RegisterRoutes,
-	RetryFailedBatch {
+	RetryBatch {
```

Comment:
```suggestion
	RestartBatch {
```

Retry implies failed, this should work for pending batches too